edk2toolext/edk2_invocable.py

```diff
@@ -46,6 +85,12 @@ def GetWorkspaceRoot(self):
         except AttributeError:
             raise RuntimeError("Can't call this before PlatformSettings has been set up!")
 
+    def GetPackagesPath(self):
+        try:
+            return self.PlatformSettings.GetPackagesPath()
+        except AttributeError:
+            raise RuntimeError("Can't call this before PlatformSettings has been set up!")
+
     def GetActiveScopes(self):
```

Comment:
-> Iterable[str]

Reply:
Do you need a tuple or iterable?

Reply:
In Edk2Invocable we are doing tuple concatenation so i need a tuple. I tried to find out what would happen if I added a tuple to an iterable …but couldn't find anything good....so decided to just keep it as a tuple.